client/src/Movies/SavedList.js

```diff
@@ -1,18 +1,19 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
 
 export default class SavedList extends Component {
-  constructor(props) {
```

Comment:
good stuff on noticing this was a useless constructor
refactoring it to a stateless functional component wouldnt have hurt anybody, right ?